android/titanium/res/menu/ti_custom_menu.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
```

Comment:
Do we want all this (the android icon, etc)? We just want to disable the normal context items, so don't need placeholder contents here, but likely just an empty menu or something.